docs/getting_started/basics/authoring.md

```diff
@@ -1,36 +1,44 @@
+
+- **Name**: The name of the plug. This must be unique among the plugs of the same direction in the same node. For example, if an input plug named *value* already exists in the node, no other input plug can have that name. Allowed plug names follow the same restrictions as variable names in Python.
+- **Type**: The type of the plug. This will determine which data type values are compatible with the plug. More details about plug types are provided [here](#TODO).
+- **Value**: The initial value of the plug on creation. 
+- **Default**: The default value of the plug. This is the value the plug will be set to when an invalid value is given or the node errors. If no default value is defined in the dialog, the default value will become the inital value of the plug.
+- **Direction**: The direction of the plug. These can be:
+    * *In*: The plug will be created at the left side of the node, which corresponds to the node's inputs. This plug will receive and/or set data to the node. 
+    * *Out*: The plug will be created at the right side of the node, which corresponds to the node's outputs. This plug will have the data processed by the node.
+    * *Through*: Two plugs with the same name will be created, one to the left side of the node as an input and another one to the right side of the node as an ouput. 
 
-![New Plug](../images/new_plug.gif)
+![Add Plug Dialog 1](../../images/create_plug.gif)
 
-You can also prompt the creation of a new plug while creating a connection. This will automatically fill in the name and type, matching the plug that the connection is coming from.
-To do that simply release your `Left-Click` mouse button over the node when creating the connection.
+### Creating New Plugs through a Connection
+Adding a new plug to a node can also be done while creating a connection. To achieve this, **drag and drop** a plug over the node where the new connection is to be added. This action will automatically prompt and populate the "*Add Plug*" dialog with the name, type, values, and direction information of the plug from which the connection was started from. Once the new plug is created, a connection between this plug and the one that was dragged will be established.
 
-![New Plug](../images/new_plug_drag_connection.gif)
+![Add Plug Dialog 2](../../images/create_plug_drag_connection.gif)
 
 ## Removing a Plug
 
-To remove a plug you can use the `Right-Click` on the plug and select "*Remove Plug*". Removing a plug will destroy all the existing connections to that plug.
+To remove a plug, `Right-Click` on the plug and select the "*Remove Plug*" option. This action will delete the plug and destroy all existing connections associated with it.
```

Comment:
We should maybe also say that a plug can be removed by left clicking on it to select it and then delete it by pressing the `delete` key

Reply:
I added a section for removing connections

Reply:
Yes sorry, I made a mistake. That should be for connections